compiler/src/test/proto/testing/rpc_name_contains_underscore.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package io.grpc.testing.underscore;
+option java_multiple_files = true;
+service NameContainsUnderscore {
+    rpc say_hello (HelloRequest) returns (HelloReply);
```

Comment:
Can we add a test case for `say_hello_again`?

Reply:
I added `say_hello_again` to proto and also unit test for this case to verify the behavior for names with multiple underscores